src/main/java/Library.java

```diff
@@ -588,4 +589,27 @@ public static String inputStreamToString(InputStream inputStream) throws IOExcep
       return stringBuilder.toString();
     }
   }
+
+  /**
+   * Create object using reflection.
+   *
+   * @param str as String
+   * @return String object
+   * @throws NoSuchMethodException if a method that does not exist at runtime.
+   * @throws IllegalAccessException <p>if an currently executing method
+   * does not have access to the definition of the specified class, field, method or constructor</p>
+   * @throws InvocationTargetException <p>InvocationTargetException is a checked exception
+   * that wraps an exception thrown by an invoked method or constructor.</p>
+   * @throws InstantiationException <p>when an method tries to create an instance of a class
+   * using the newInstance method in class Class.</p>
+   */
+  public static String createObject(String str)
+          throws NoSuchMethodException,
+          IllegalAccessException,
+          InvocationTargetException,
+          InstantiationException {
+    var stringClass = String.class;
+    var stringConstructor = stringClass.getConstructor(String.class);
+    return stringConstructor.newInstance(str);
```

Comment:
Although the method name implies that it can create any object, it always seems to return instance of String class. We could make it more general by getting the class name from the user as parameter and using `Class.forName(...` to get the class.